cmd/ps.go

```diff
@@ -120,10 +121,16 @@ func printProcesses(appID string, processes []api.Pods) {
 
 	fmt.Printf("=== %s Processes\n", appID)
 
-	for psType, procs := range psMap {
+	var psTypes []string
+	for psType := range psMap {
+		psTypes = append(psTypes, psType)
+	}
+	sort.Strings(psTypes)
+
+	for _, psType := range psTypes {
 		fmt.Printf("--- %s:\n", psType)
 
-		for _, proc := range procs {
+		for _, proc := range psMap[psType] {
 			fmt.Printf("%s %s (%s)\n", proc.Name, proc.State, proc.Release)
```

Comment:
Just realized this is V2 (which I am not particularly familiar with; and in fact I did not test this code with a real cluster), and I believe I saw that V2 has "cryptic" names for instances which indeed might not be sorted....

I just took the same code that I changed for V1 version (which I tested) and copied over to V2, but in fact behaviour might indeed be different